decidim-conferences/app/commands/decidim/conferences/admin/update_conference.rb

```diff
@@ -127,6 +130,34 @@ def schedule_upcoming_conference_notification
             .set(wait_until: (@conference.start_date - 2.days).to_s)
             .perform_later(@conference.id, checksum)
         end
+
+        def participatory_processes(conference)
+          @participatory_processes ||= conference.participatory_space_sibling_scope(:participatory_processes).where(id: @form.participatory_processes_ids)
+        end
+
+        def link_participatory_processes(conference)
+          conference.link_participatory_spaces_resources(participatory_processes(conference), "included_participatory_processes")
+        end
+
+        def assemblies(conference)
+          @assemblies ||= conference.participatory_space_sibling_scope(:assemblies).where(id: @form.assemblies_ids)
+        end
+
+        def link_assemblies(conference)
+          conference.link_participatory_spaces_resources(assemblies(conference), "included_assemblies")
+        end
+
+        def consultations(conference)
+          @consultations ||= conference.participatory_space_sibling_scope(:consultations)
+                                       .includes(:consultation)
+                                       .where(decidim_consultation_id: @form.consultations_ids)
```

Comment:
Same problem, I think we know too much about how consultations work. What happens if the app doesn't have `decidim-consultations`? It's an optional gem, and it's not bundled with `decidim` metagem 😕

Reply:
So the principal concern here is: why can't you use a simple `where(id: @form.consultation_ids)` here? Why all these includes and collects?

Reply:
Is more or less what I say before. The Participatory Space in Consultations mdoule is Question. FI we need to related Conference with Consultation we need to do something like that

Reply:
Although I agree it's weird, both Consultations and Questions are participatory spaces, how come if you ask for consultations you get questions?

Reply:
The issue seems to be here:

https://github.com/decidim/decidim/blob/master/decidim-consultations/lib/decidim/consultations/participatory_space.rb#L3-L27

I don't know why questions are the space and consultations the resource.

Reply:
I'd suggest dropping all integration with consultations then.

Reply:
Correct, that is the problem @oriolgual... I do not know how we can do in this case.

Reply:
Maybe @jsperezg can help us here.

Reply:
What happens if you change the `model_class_name` to Consultation @isaacmg410?

Reply:
Will many things break up? Then we will change the `register_resource` to `Question`?

Reply:
I'm going to try it at my machine.

Reply:
I changed:

```ruby
participatory_space.model_class_name = "Decidim::Consultations::Question"
```

To 

```ruby
participatory_space.model_class_name = "Decidim::Consultation"
```

And

```ruby
Decidim::Consultations::Question.where(organization: organization)
```

To

```ruby
Decidim::Consultation.where(organization: organization)
```

At 
https://github.com/decidim/decidim/blob/master/decidim-consultations/lib/decidim/consultations/participatory_space.rb

And all tests pass and the navigation seems to work OK. And `conference.participatory_space_sibling_scope(:consultations)` return consultations as expected.

Reply:
I will try also, and if all is okey. Go for it?